common/hashicorp-vault/values.yaml

```diff
@@ -47,5 +36,6 @@ vault:
         # get vault.vault.svc/vault.vault.svc.cluster.local
         termination: "reencrypt"
     image:
-      repository: "registry.connect.redhat.com/hashicorp/vault"
```

Comment:
To be consistent, I think the image should default to the non-sgx enabled one, and override to the sgx-enabled one from the caller. (It's fine to document where to get the sgx enabled version in the values file)

Reply:
I reverted it to the default image